.evergreen/docker/ubuntu20.04/base-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+
+rm -f $DRIVERS_TOOLS/results.json
+cd $DRIVERS_TOOLS/.evergreen
+bash run-orchestration.sh
+
+# Change permissions of files we have created.
+cd $DRIVERS_TOOLS
+files="results.json uri.txt .evergreen/mongo_crypt_v1.so .evergreen/mo-expansion.yml"
+for fname in $files; do
+    chown --reference=action.yml $fname
+    chmod --reference=action.yml $fname
+done
```

Comment:
```suggestion
files=(results.json uri.txt .evergreen/mongo_crypt_v1.so .evergreen/mo-expansion.yml)
chown --reference=action.yml "$files[@]"
chmod --reference=action.yml "$files[@]"
```
Both `chown` and `chmod` support multiple files as arguments, so we should be able to elide the loop (use of an array is motivated by [SC2086](https://www.shellcheck.net/wiki/SC2086)).